modules/ding_frontend/ding_frontend.pages_default.inc

```diff
@@ -562,7 +562,14 @@ search/node',
       'override_title_text' => '',
       'override_title_heading' => 'h2',
     );
-    $pane->cache = array();
+    $pane->cache = array(
+      'method' => 'lazy',
```

Comment:
I would think this required `lazy_pane` to be added as a dependency for `ding_frontend`?